core-rust/core-api-server/src/core_api/handlers/status_network_configuration.rs

```diff
@@ -121,6 +121,11 @@ fn to_api_address_type(hrp_set: &HrpSet, entity_type: EntityType) -> models::Add
             models::EntityType::Clock,
             extract_length(ComponentAddress::Clock),
         ),
+        EntityType::AccessControllerComponent => (
```

Comment:
This, and all the three identity components, need adding to `ALL_ENTITY_TYPES` above